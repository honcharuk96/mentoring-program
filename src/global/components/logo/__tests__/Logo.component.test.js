import React from 'react';
import LogoComponent from '../logo.component';
import ShallowRenderer from 'react-test-renderer/shallow';
import 'jest-styled-components';

const renderer = new ShallowRenderer();
describe('LogoComponent', function () {
   const logoComponent = renderer.render(<LogoComponent/>)
    it('snapshot', () => {
        expect(logoComponent).toMatchSnapshot()
    })
});
