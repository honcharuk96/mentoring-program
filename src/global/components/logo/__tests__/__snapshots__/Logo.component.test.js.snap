// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoComponent snapshot 1`] = `
<logostyled__Logo>
  <globalStyles__Bold>
    netflix
  </globalStyles__Bold>
  roulette
</logostyled__Logo>
`;
